src/ZeroNuFit.jl

```diff
@@ -70,15 +70,16 @@ Parameters:
     else
         @info "... we load already existing fit results"
         samples = bat_read(joinpath(config["output_path"],"mcmc_files/samples.h5")).result
-        prior,par_names=build_prior(partitions,part_event_index,config=config,stat_only=config["stat_only"])
-
+        prior,_,_,par_names = get_stat_blocks(partitions,events,part_event_index,config=config,stat_only=config["stat_only"]) 
+        
     end
     
     @info bat_report(samples)
     
     _,_,posterior,_ = get_stat_blocks(partitions,events,part_event_index,config=config,stat_only=config["stat_only"]) 
     
-    save_outputs(partitions, events, part_event_index, samples, posterior, config,priors=prior,par_names=par_names)
+    hier = config["bkg"]["correlated"]
+    save_outputs(partitions, events, part_event_index, samples, posterior, config,priors=prior,par_names=par_names,hier=hier)
```

Comment:
we already pass the config, maybe you can defined "hier" directly in "save_outputs"? so we dont increase too much the number of things to pass to that fucntion @tdixon97

Reply:
sure